src/View/OutputView.js

```diff
@@ -0,0 +1,37 @@
+const { Console } = require('@woowacourse/mission-utils');
+const { MESSAGE } = require('../libs/Constant');
+
+/**
+ * 사용자에게 게임 진행 상황과 결과를 출력하는 역할을 한다.
+ */
+const OutputView = {
+  printMessage(message) {
+    Console.print(message);
+  },
+
+  /**
+   * 현재까지 이동한 다리의 상태를 정해진 형식에 맞춰 출력한다.
+   * <p>
+   * 출력을 위해 필요한 메서드의 인자(parameter)는 자유롭게 추가하거나 변경할 수 있다.
+   */
+  printMap([upBridge, downBridge]) {
+    Console.print(`[ ${upBridge.join(' | ')} ]`);
+    Console.print(`[ ${downBridge.join(' | ')} ]`);
```

Comment:
중복 코드 같아 보여서 이렇게 표현하는 건 어떻게 생각하시나요! 🤔

```suggestion
    [upBridge, downBridge]
        .forEach((bridge) => Console.print(`[ ${bridge.join(' | ')} ]`));
```

Reply:
오! 너무 좋은 방법이네요👍
코드가 더욱 간결해 보입니다:)